src/commands/indexer/index.ts

```diff
@@ -1026,12 +1027,16 @@ export default class Indexer extends HealthCheck {
             const response = await this.apiService.updateCrossChainTransactionStatus(input)
             this.networkMonitor.structuredLog(
               network,
-              this.apiColor(`API: Cross chain transaction ${jobHash} mutation response ${JSON.stringify(response)}`),
+              this.apiColor(
+                `API: Cross chain message transaction ${jobHash} mutation response ${JSON.stringify(response)}`,
```

Comment:
I wonder if these logs with `JSON.stringify` will go on multiple lines in DD and look like errors